client/src/modules/purchases/modals/search.tmpl.html

```diff
@@ -15,94 +15,80 @@
     <uib-tabset>
       <uib-tab index="0" heading="{{ 'FORM.LABELS.SEARCH_QUERRIES' | translate}}" data-custom-filter-tab>
         <div class="tab-body">
+          <!-- Reference -->
           <div class="form-group" ng-class="{ 'has-error' : ModalForm.$submitted && ModalForm.reference.$invalid }">
             <label class="control-label" translate> FORM.LABELS.REFERENCE </label>
-            <bh-clear on-clear="ModalCtrl.clear('reference')"></bh-clear>
+            <bh-clear on-clear="$ctrl.clear('reference')"></bh-clear>
 
-            <input type="text" class="form-control" name="reference" ng-model="ModalCtrl.params.reference">
+            <input type="text" class="form-control" name="reference" ng-model="$ctrl.searchQueries.reference">
             <div class="help-block" ng-messages="ModalForm.reference.$error" ng-show="ModalForm.$submitted">
               <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
             </div>
           </div>
 
           <!-- supplier -->
-          <div class="form-group">
-            <label class="control-label" translate>FORM.LABELS.SUPPLIER</label>
-            
-            <bh-clear on-clear="ModalCtrl.clear('supplier_uuid')"></bh-clear>
-
-            <ui-select name="supplier" ng-model="ModalCtrl.params.supplier_uuid">
-              <ui-select-match>
-                <span>{{$select.selected.display_name}}</span>
-              </ui-select-match>
-              <ui-select-choices ui-select-focus-patch repeat="supplier.uuid as supplier in ModalCtrl.suppliers | filter:$select.search">
-                <span ng-bind-html="supplier.display_name | highlight:$select.search"></span>
-              </ui-select-choices>
-            </ui-select>
-          </div>
-
+          <bh-supplier-select
+            supplier-uuid="$ctrl.searchQueries.supplier_uuid"
+            on-select-callback="$ctrl.onSelectSupplier(supplier)"
+            required="false">
+            <bh-clear on-clear="$ctrl.clear('supplier_uuid')"></bh-clear>
+          </bh-supplier-select>
+    
           <!-- user -->
           <bh-user-select
-            user-id="ModalCtrl.bundle.user_id"
+            user-id="$ctrl.searchQueries.user_id"
             name="user_id"
-            on-select-callback="ModalCtrl.onSelectUser(user)">
-            <bh-clear on-clear="ModalCtrl.clear('user_id')"></bh-clear>
-          </bh-user-select> 
-
+            on-select-callback="$ctrl.onSelectUser(user)">
+            <bh-clear on-clear="$ctrl.clear('user_id')"></bh-clear>
+          </bh-user-select>
 
           <!-- status -->
           <div class="form-group">
             <p class="control-label" style="margin-bottom:5px;">
               <strong translate>FORM.LABELS.STATUS</strong>
             </p>
 
-            <label class="radio-inline">
-              <input type="radio" id="is_confirmed" name="state" ng-model="ModalCtrl.setState" value="is_confirmed">
+            <label class="checkbox-inline">
```

Comment:
We had a discussion about this [in this comment](https://github.com/IMA-WorldHealth/bhima-2.X/pull/1907#discussion_r131340490) and in [this review](https://github.com/IMA-WorldHealth/bhima-2.X/pull/1907#issuecomment-320877860).

For checkboxes to work, you would expect there to be a case where a purchase order could be both `is_received` and `is_cancelled` (since the inputs are not exclusive).  If this is the case, the SQL should be updated to reflect that.  Otherwise, they should be radio buttons.

As discussed yesterday, we might be removing these states altogether.  I propose you leave these are radio buttons until we figure out how we want to use these properties.